_posts/2022-02-19-newpipe-0.22.0.md

```diff
@@ -0,0 +1,155 @@
+
+ -[@XiangRongLin](https://github.com/XiangRongLin) ensured that search history entries in tests are in the correct order, fixing the sometimes-failing-sometimes-passing CI pipeline.
+
+- [@Stypox](https://github.com/Stypox) fixed many different crashes where the code expected the player's queue to be not `null` but it was, indeed, `null`.
+
+- [@Cheechaii](https://github.com/Cheechaii) introduced #respect into the app. Now the player respects display cutouts (like notches) when playing in multi-window mode, whereas previously, black bars were shown.
+
+- Speaking of hashtags, [@Jaspann](https://github.com/Jaspann) made their first contribution to a public open source project (Congratulations!): They fixed hashtags that use non-English characters. Thank you!
+
+- [@Jaspann](https://github.com/Jaspann), in their second contribution, fixed showing the PeerTube-specific sub-channel icon on non-PeerTube services, where it makes no sense.
+
+- [@TobiGr](https://github.com/TobiGr) fixed a link in the English README linking to the Portuguese README. Not really their first/second contribution to the project, a welcome one nonetheless
+
+- Sometimes, the keyboard would stop working when using the search field after the video page was closed, no matter how hard you smashed the keys. [@litetex](https://github.com/litetex) has fixed this to prevent you from damaging your screen.
+
+- When items were added to the queue using the long-press menu in lists, it was clear that there was some info missing when the queue was opened. This is because the full stream info was loaded only when its page was opened. Thanks to [@Douile](https://github.com/Douile), now the complete stream info is loaded when it is added to the queue from a list.
+
+- If you tried to share a playlist while it was still loading (you maniac) it would cause the app to crash, which has been fixed by [@TiA4f8R](https://github.com/TiA4f8R).
+
+- [@Stypox](https://github.com/Stypox) added the app icon to the fastlane metadata folder, so that the F-Droid app and website start showing the icon again.
+
+- Ooh. This is a tricky one. If a stream other than the first one in the queue was playing, and its resolution was changed, or the player type was changed (e.g. popup -> main player), the app would suddenly abandon that stream and start playing the first stream in the queue instead. This weird bug, caused by an ExoPlayer update, has been fixed by [@TiA4f8R](https://github.com/TiA4f8R).
+
+- If you tried to view the channel page of an item in the queue whose channel page you hadn't already viewed before, you would get a network error, which has been fixed by [@ktprograms](https://github.com/ktprograms).
+
+## Localisation
```

Comment:
Where is https://github.com/TeamNewPipe/NewPipe/pull/7694?

Reply:
Quite sure that wasn't in the draft, so I want to point the blame to someone else, but yes. Needs to be added ofc.

Reply:
> so I want to point the blame to someone else

That would be me. xD